app/controllers/completed_submissions_controller.rb

```diff
@@ -7,6 +7,12 @@ def index
 	end
 
 	def print
+		if params[:print_top]
+			params[:printer] = params[:printer_top]
+		elsif params[:print_bottom]
+			params[:printer] = params[:printer_bottom]
+		end
```

Comment:
Both params (print_top and print_bottom) are going to be sent always because they are in the same form. What would happen if the user change both printing selects and clicks on print?